monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md

```diff
@@ -57,46 +57,46 @@ Monero — это слово на языке эсперанто. Создате
 
 **Доступность майнинга -** принять участие в майнинге может любой человек, у которого есть подключённое к сети устройство или веб-браузер.
 
 ### ЗНАЧЕНИЕ ДЛЯ РЕАЛЬНОГО МИРА И ВОЗМОЖНОСТИ ИСПОЛЬЗОВАНИЯ
 
-Поскольку Monero является безопасной и не знающей границ монетой, а комиссии за проведение транзакций малы, люди могут использовать эти деньги, не опасаясь коррумпированных правительств и краха банков. Это укрепляет экономическое положение людей в странах с деспотическими режимами или депрессивной экономикой. Приватность финансовой истории защищает потребителей и компании от манипулирования ценами, недобросовестного использования цепи поставки, **[экономической дискриминации](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** и других подобных вещей.
+Поскольку Monero является безопасной и не знающей границ монетой, а комиссии за проведение транзакций малы, люди могут использовать эти деньги, не опасаясь коррумпированных правительств, краха банков и возможность вести бизнес без конкурентов, отслеживающих вашу личную информацию. Это укрепляет экономическое положение людей в странах с деспотическими режимами или депрессивной экономикой. Приватность финансовой истории защищает потребителей и компании от манипулирования ценами, недобросовестного использования цепи поставки, **[экономической дискриминации](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** и других подобных вещей в высококонкурентной деловой среде.
 
 **[Monero является единственной криптовалютой](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)**, которая обладает всеми свойствами, делающими её полностью взаимозаменяемой, децентрализованной электронной валютой.
 
 ### ТЕХНИЧЕСКИЕ ОСНОВЫ
 
-_(11/14/2018)_
+_(По состоянию на 02/01/2019)_
 
-+ **Количество активных нод:** 1,756 (Источник: https://monerohash.com/nodes-distribution.html)
++ **Количество активных нод:** 1,564 (Источник: https://monerohash.com/nodes-distribution.html)
 
-+ **Хешрейт сети:** 435.17 MH/s
++ **Хешрейт сети:** 663.2 MH/s
 
-+ **Количество транзакций в час:** 178 (Средний показатель за 30 дней)
++ **Количество транзакций в час:** 149 (Средний показатель за 30 дней)
 
-+ **Ядра CPU, обеспечивающие безопасность сети:** 14,499,178
++ **Ядра CPU, обеспечивающие безопасность сети:** 22,234,297
 
-+ **Monero в обращении:** 16,570,126 XMR (Приблизительно)
++ **Monero в обращении:** 16,758,217 XMR (Приблизительно)
 
-+ **Рыночная капитализация:** $1,503,928,296 USD (~0.77% от общего объема капитализации)
++ **Рыночная капитализация:** $743,944,945 USD (~0.77% от общего объема капитализации)
```

Comment:
We found the information we had posted was incorrect and thanks to @el00ruobuob we were able to fix it. This is the string with the correct information.

**Market Capitalization:** $732,932,071 USD (~0.65% of total cryptocurrency market cap)

Reply:
fix